CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ add_dependencies(${PROJECT_NAME}
     )
 
 install(DIRECTORY launch
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
```

Comment:
I thought the original was good, the launch files go in `.../share/<package>/launch` folder right?